mmengine/config/config.py

```diff
@@ -1605,6 +1606,12 @@ def merge_from_dict(self,
             Config._merge_a_into_b(
                 option_cfg_dict, cfg_dict, allow_list_keys=allow_list_keys))
 
+    @staticmethod
+    def diff(cfg1, cfg2) -> str:
```

Comment:
It would be convenient if `cfg` can be a file path. We can check if `cfg` is a string, calling `fromfile` to obtain the config.

Reply:
Added in newest commit